graphql.go

```diff
@@ -386,6 +389,28 @@ func newError(code string, err error) Error {
 	}
 }
 
+type ErrResponseStatusNotOK struct {
```

Comment:
```suggestion
type NetworkError struct {
```

We should follow the naming of other GraphQL clients such as [Apollo](https://www.apollographql.com/docs/react/api/link/apollo-link-error/#networkerror).